Public/Invoke-AtomicTest.ps1

```diff
@@ -32,7 +32,7 @@ function Invoke-AtomicTest {
             ValueFromPipelineByPropertyName = $true,
             ParameterSetName = 'technique')]
         [ValidateNotNullOrEmpty()]
-        [String]
+        [String[]]
```

Comment:
Had to convert it into array since there might a comma in short form (T1554.004-1,2,3)